src/09-word-problems.js

```diff
@@ -40,7 +51,13 @@ function applyDiscount(priceInCents, age, hasMembership) {}
     getCartTotal(cart);
  *  //> "$30.00"
  */
-function getCartTotal(products) {}
+function getCartTotal(products) {
+  let total = 0;
+  if (products.name) {
```

Comment:
What is `products`? Does it have a `.name` property? Is it an object? An array of objects?